src/LuigisBoxBundle/ValueObject/SearchUrlBuilder.php

```diff
@@ -125,6 +125,10 @@ public function addFilter(string $key, string $value): self
 
     public function setFilters(array $filters): self
     {
+        if (\count($this->filters ?? []) > 0) {
+            throw new \LogicException('You already have the filters set. Use resetFilters() method to clear them first.');
```

Comment:
Why this is important? Usually setters override old values. I don't get why reset is required here.

Reply:
becouse this is builder. I want to know when I overwrite filters because when I wrote
$qb->setQuery()
->addFilter('')
->addFilter('');

//some other code here
$qb->setFilters([]);

I can do it unintentionally